### ISSUE TITLE
r740 LUA error

### ISSUE THREAD

**Issue by dmsmith111** [arkayenro/arkinventory#1061]
Updated to r740 and am now getting this error:

```
21x ArkInventory\ArkInventory-30805.lua:11010: ArkInventory\ArkInventoryLDB.lua:211: attempt to index field 'style' (a number value)
[C]: ?
ArkInventory\ArkInventory-30805.lua:11010: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>

Locals:
(*temporary) = "ArkInventory\ArkInventoryLDB.lua:211: attempt to index field 'style' (a number value)"

```

and

`23x ArkInventory\ArkInventory-30805.lua:11010: ArkInventory\ArkInventoryLDB.lua:211: attempt to index field 'style' (a number value)
[C]: ?
ArkInventory\ArkInventory-30805.lua:11010: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>

Locals:
(*temporary) = "ArkInventory\ArkInventoryLDB.lua:211: attempt to index field 'style' (a number value)"
`

**Comment by arkayenro** [comment]
741 alpha should be up soon which should hopefully fix that

**Comment by arkayenro** [comment]
fixed in 3.08.05
